README.md

```diff
@@ -1,38 +1,19 @@
 # SIG Release
 
-## Mission
-- Production of high quality Kubernetes releases on a reliable schedule
-- Ensure there is a [consistent group of community members][1.6-retro] in place to support the release process
-- Provide guidance and tooling to facilitate the production of automated releases
-- Serve as a tightly integrated partner with other SIGs to empower SIGs to integrate their repositories into the release process
+## Charter
 
-## Join us!
-- [Google group](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
-- [Slack channel](https://kubernetes.slack.com/messages/C2C40FMNF/)
-- [Events and meetings calendar](https://calendar.google.com/calendar/embed?src=coreos.com_regcvcrgvq98lua2ikijg1g1uk%40group.calendar.google.com&ctz=America/Los_Angeles)
```

Comment:
Hey, this deletes all meeting and contact information, and that meeting information is NOT replicated in the charter.  IMHO, we should leave meeting and contact information in the README.